src/main/java/io/axoniq/axonserver/connector/admin/AdminChannel.java

```diff
@@ -107,6 +108,40 @@
      */
     CompletableFuture<Void> mergeEventProcessor(String eventProcessorName, String tokenStoreIdentifier);
 
+    /**
+     * Request to balance the load for the given {@code eventProcessorName} within the connected client.
+     * Returns a {@link CompletableFuture} that completes when the request has been received by Axon Server.
+     * Note that this doesn't imply that the processor is balanced, but only that the request has been properly
+     * delivered.
+     *
+     * @param eventProcessorName   the name of the event processor to balance the load for
+     * @param tokenStoreIdentifier the token store identifier of the processor to balance the load for
+     * @param strategy             the balancing strategy to use
+     * @return a {@link CompletableFuture} that completes when the request is delivered to Axon Server
+     */
+    CompletableFuture<Void> loadBalanceEventProcessor(String eventProcessorName, String tokenStoreIdentifier, String strategy);
+
+    /**
+     * Request to auto balance the load for the given {@code eventProcessorName} within the connected client.
+     * Returns a {@link CompletableFuture} that completes when the request has been received by Axon Server.
+     * Note that this doesn't imply that the processor is balanced, but only that the request has been properly
+     * delivered.
+     *
+     * @param eventProcessorName   the name of the event processor to balance the load for
+     * @param tokenStoreIdentifier the token store identifier of the processor to balance the load for
+     * @param strategy             the balancing strategy to use
+     * @return a {@link CompletableFuture} that completes when the request is delivered to Axon Server
+     */
+    CompletableFuture<Void> autoLoadBalanceEventProcessor(String eventProcessorName, String tokenStoreIdentifier, String strategy);
+
+
+    /**
+     * Returns all available load balance strategies registered to AxonServer.
+     *
+     * @return the list of all available load balance strategies registered to AxonServer.
+     */
+    CompletableFuture<List<LoadBalancingStrategy>> getBalancingStrategies();
```

Comment:
I can make assumption here, but I think it's better to ask: how are you going to use this method exactly?

Reply:
Its in the test:

```
     CompletableFuture<Void> accepted = adminChannel.getBalancingStrategies()
                .thenCompose(balancingStrategies -> adminChannel.loadBalanceEventProcessor("processor", "tokenStore", balancingStrategies.get(0).getStrategy()));
```